### ISSUE TITLE
Simple documentation typo

### ISSUE THREAD

**Issue by thebluejay** [ember-cli/eslint-plugin-ember#240]
The documentation for the no-jquery rule  ([`docs/rules/no-jquery.md`](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-jquery.md)) has a confusing typo, where the second example of correct code for this rule, is actually incorrect.

Currently the example is:

```js
import $ from 'jquery';

export default Component.extend({
  click() {
    document.body.classList.add('expanded');
  }
});
```

When it should be lacking the jQuery import like so:

```js
export default Component.extend({
  click() {
    document.body.classList.add('expanded');
  }
});
```

**Comment by twokul** [comment]
@thebluejay do you mind submitting a quick PR to address the issue?
